extensions/src/main/java/org/pentaho/platform/plugin/services/metadata/PentahoMetadataDomainRepository.java

```diff
@@ -675,6 +823,82 @@ private void internalReloadDomains() {
     }
   }
 
+  /**
+   * POC - DEBUG
```

Comment:
TODO remove DEBUG comment in javadoc getFileMetadataHelper(

Reply:
see commit: https://github.com/pentaho/pentaho-platform/pull/5124/commits/2277f57c5f89124e2e5b11b6591f46694b98350d